requirements.txt

```diff
@@ -19,4 +19,5 @@ django-nested-admin==3.0.2
 django-geojson==2.9.0
 django-leaflet==0.18.0
 django-ckeditor==5.0.3
-django-autoslug==1.9.3
\ No newline at end of file
+django-autoslug==1.9.3
+django-allauth
```

Comment:
Might be good idea to pin the `django-allauth` too. Pinned versions makes things more robust.